.github/workflows/publish.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and Publish Client to PyPI
+on:
+  release:
+    types: [published]
+jobs:
+  build-and-publish-client:
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      # url: TODO: Add URL to PyPI environment
+    permissions:
+      id-token: write
+    defaults:
+      run:
+        working-directory: adit-client
```

Comment:
not adit-client

Reply:
I guess you can remove that working directory completely as "uv build" must be called in the root directory of the repo (which I guess is the default).
(In ADIT adit-client is in a separate directory having it's own pyproject.toml)